src/interfaces/chain.h

```diff
@@ -327,6 +327,15 @@ class Chain
         virtual void chainStateFlushed(ChainstateRole role, const CBlockLocator& locator) {}
     };
 
+    //! Options specifying which chain notifications are required.
```

Comment:
https://github.com/bitcoin/bitcoin/pull/32694/commits/6f1392cc42cde638773f2b697d7d2c58abcdc860 (_indexes, refactor: Remove remaining CBlockIndex* uses in index Rewind methods_)

---

nit:

```suggestion
    //! Options specifying data required for chain notifications.
```